src/commands/price.ts

```diff
@@ -105,11 +105,11 @@ export class PriceCommand extends Command {
 		return this.#logger;
 	}
 
-	// TODO: i18n
-	private vendorFormats = {
-		tcgplayer: (price: number): string => `$${price.toFixed(2)}`,
-		cardmarket: (price: number): string => `€${price.toFixed(2)}`,
-		coolstuffinc: (price: number): string => `$${price.toFixed(2)}`
+	// specify Record type to avoid repeating type information on each property
+	private vendorFormats: Record<string, (locale: Locale) => Intl.NumberFormat> = {
```

Comment:
key here should be the `vendorId` type that already exists in the file but otherwise all good!